apps/collections/api/serializers.py

```diff
@@ -1,62 +1,26 @@
 from apps.collections.models import Collection
 from apps.entities.api.serializers import EntitySerializer
-from apps.entities.models.entity import Entity
 from apps.propositions.api.serializers import PropositionSerializer
-from apps.propositions.models.proposition import Proposition
 from apps.quotes.api.serializers import QuoteSerializer
-from apps.quotes.models.quote import Quote
 from apps.sources.api.serializers import SourceSerializer
-from apps.sources.models.source import Source
 from core.models.serializers import SluggedModelSerializer
 
 
 class CollectionSerializer(SluggedModelSerializer):
     """Serializer for collection."""
 
-    propositions_serialized = PropositionSerializer(
```

Comment:
We need both fields for this to work correctly. Non-serialized version accepts for writes with ids and serialized versions were being used as read-only. With these changes, it's now not working as expected because the api now expects whole sources, quote etc objects instead of existing ids.

Reply:
@alashow, I'm confused because I don't see `propositions_serialized` being removed in the file diff for `apps/collections/api/serializers.py` at https://github.com/ModularHistory/modularhistory/pull/900/files

It looks to me like I just renamed the serializers without removing anything...